src/layout/section/cardsSection/CardsSection.tsx

```diff
@@ -0,0 +1,21 @@
+import React from 'react'
+import cc from 'classcat'
+
+import { TripCardProps } from 'tripCard'
+import { QrCardProps } from 'qrCard'
+
+type CardsProps = TripCardProps | QrCardProps
```

Comment:
Why using the props and not the component?

```
type CardType = TripCard | QrCard
 ```

Then 

```
readonly children: CardType[] | CardType
```

Reply:
It seems not permitted

`'TripCard' refers to a value, but is being used as a type here.ts(2749)
Property 'children' of exported interface has or is using private name 'TripCard'.ts(4033)`